tools-adoption.md

```diff
@@ -0,0 +1,23 @@
+# Tools across our projects
+
+This page is supposed to track the adoption of the tools that want to use across our projects.
+
+## Goals
+
+- Migrate from Travis CI to GitHub Actions
+- Migrate GitHub wiki documentation to Docusaurus
+- Have a deployable version of each app. E.g.: Heroku or GitHub Pages or mobile apk
+
+## Tracking
+
+| Project | GitHub Actions | Docusaurus | Heroku | GitHub Pages |
+|-|:-:|:-:|:-:|:-:|
+| anitab-org.github.io | ✅ | ❌ | N/A | ✅ |
+| mentorship-backend | ✅ | ❌ | ✅ | N/A |
+| mentorship-android | ❌ | ❌ | N/A | ✅ |
+| stem-diverse-tv | ❌ | ❌ | ✅ | N/A |
+| stem-diverse-tv-cms | ❌ | ❌ | ❌ | N/A |
+| bridge-in-tech-backend | ❌ | ❌ | ✅ | N/A |
+| bridge-in-tech-web | ❌ | ❌ | ❌ | N/A |
```

Comment:
@isabelcosta, both BIT backend and frontend actually already have the Heroku remote server dedicated to each repo. They just cannot be utilised since the Heroku free tier service limitation not allowing them to work as expected. So, I'm not sure how we should tag this status (with or without tick). 🤔

Reply:
Also, if an issue is currently open for the tool, should we use a different tag (either "In progress" or directly put the issue number with the link so it's easy to refer to it and monitor the progress)?

Reply:
@mtreacy002 Good idea. I suggest to use 🚧 or 🕙 for showing `in progress` items

Reply:
Yap, that sounds good! I actually thought about that, but I also thought that having a cross would help see it as a issue. We don't want to keep something like this as in progress for too long, so I was thinking of cross to make sure it appears has an issue. I can put that :)